lib/macho/load_commands.rb

```diff
@@ -171,6 +27,33 @@ def self.new_from_bin(view)
       self.new(view, *bin.unpack(format))
     end
 
+    # Creates a new (viewless) command corresponding to the symbol provided
+    # @param cmd_sym [Symbol] the symbol of the load command being created
+    # @param args [Array] the arguments for the load command being created
+    # @note this does not currently work. DO NOT USE IT.
+    def self.create(cmd_sym, *args)
+      raise LoadCommandNotCreatableError.new(cmd_sym) unless CREATABLE_LOAD_COMMANDS.include?(cmd_sym)
+
+      klass = MachO.const_get "#{LC_STRUCTURES[cmd_sym]}"
+      cmd = LOAD_COMMAND_CONSTANTS[cmd_sym]
+      cmdsize = klass.bytesize
+
+      # this is a hack, since the only currently creatable load commands are
+      # ones that contain strings.
+      args.each do |arg|
+        if arg.is_a?(String)
+          cmdsize += arg.bytesize
+        end
+      end
```

Comment:
Let's drop this hack. This belongs into the serialization, where we'll have enough information to compute this properly.